paddle/operators/name_convention.md

```diff
@@ -0,0 +1,15 @@
+## Operator Name Convention
+
+To make the operator document itself more clear. we recommend operator names observe the listing conventions.
+
+### Input/Output names
+
+* Variable name is prefer uppercase. e.g. `X`, `Y`. But when the variable is tensor, its name should lowercase. e.g. `matrix`, to discriminate with otherone.
```

Comment:
if the name has multiple letters, such as `all`, what should we name it `All` or `ALL`?